lib/marked.esm.js

```diff
@@ -587,9 +583,9 @@ class Tokenizer {
       for (i = 0; i < l; i++) {
         this.lexer.state.top = false;
         list.items[i].tokens = this.lexer.blockTokens(list.items[i].text, []);
-        if (list.items[i].tokens.some(t => t.type === 'space')) {
+        if (!list.loose && list.items[i].tokens.some(t => t.type === 'space')) {
           list.loose = true;
-          list.items[i].loose = true;
+          // list.items[i].loose = true;
```

Comment:
Remove this comment too?

Reply:
These are just the compiled files which are out of date. They are not present in the actual source code so I don't think they matter here (they get regenerated when we merge, right?), but I will recompile now to see what happens.

Reply:
There. Lib files were updated.